src/components/Service.jsx

```
@@ -1,11 +1,102 @@
+        >
+          <thead>
+            <tr>
+              <th>Service</th>
+              <th>Price (VND)</th>
+            </tr>
+          </thead>
+          <tbody>
+            {service.map((item) => (
+              <>
+                <tr>
+                  <td style={{paddingLeft: "0.25em"}}>{item.orderServiceDetailName}</td>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)